### PR TITLE
chore(mobile): simplify qr code scanner

### PR DESCRIPTION
**Pull request by Nodonisko**
Not tested yet on real device.

### CODE REVIEW

File: suite-native/module-accounts-import/src/components/Camera.tsx [hunk 1]
```diff
@@ -5,11 +5,10 @@ import { Linking, View } from 'react-native';
 import { BarcodeFormat, useScanBarcodes } from 'vision-camera-code-scanner';
 
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
-import { XpubAddress } from '@suite-common/wallet-types';
 import { Text } from '@suite-native/atoms';
 
 export interface CameraProps {
-    onResult: (xpubAddress?: XpubAddress) => void;
+    onResult: (qrCodeValue: string) => void;
```
Comment (Nodonisko) [inline suite-native/module-accounts-import/src/components/Camera.tsx:11]: Since user can scan any kind of QR code I would rather not imply it always xpub

**Review by juriczech — APPROVED** [review]
Apart from Dan's comment its fine. One nitpick I would suggest is to add comment to this: 
Just comment why you pick the first item in array just so developer doesn't event have to think about it. But it's just a suggestion and maybe it's unnecessary.
`const barcode = barcodes?.[0]?.displayValue;`

**Comment by bouchja1** [comment]
FYI @Nodonisko, @juriczech It is not working on a real device... :( I just tried it.

**Review by bouchja1 — CHANGES_REQUESTED** [review]
It is not working on a real device. Please, fix it.

**Pull request closed without merging**